contents/handbook/strategy/strategy.md

```diff
@@ -4,53 +4,165 @@ sidebar: Handbook
 showTitle: true
 ---
 
-> PostHog's mission is to increase the number of successful products in the world.
+## TL;DR
+Our mission is to **_“Increase the number of successful products in the world”_**. Our first step on their journey is to prove product market fit with 5 reference customers. Today we're closest to PMF with products which need to self-host and are in the growth phase of their lifecycle; they need us to absolutely "Nail Funnels" in order to make their products successful.
 
-##Companies we build for
+### Context
 
-PostHog is *used* by companies of every scale - hobbyists, startups, scale ups and enterprises.
+We’ve grown a lot, but it’s clear that the majority of our users use us *despite* a bad experience - bugs, instability, lots of maintenance or confusing UX. This is a great problem to have. It means we’re solving a hair on fire problem - product analytics that you can self host.
```

Comment:
Is it accurate to say that most users have a bad experience across all types? I suspect we're possibly being a bit _too_ self-critical - we don't tend to get a huge number of support queries vs. the number of active users we have. 

Perhaps 'it's clear that a significant proportion of our self-hosted users use us...' would be more accurate? (Happy to be overruled on this though if the data supports this though - I agree we shouldn't sugar coat the extent of any issues).